### PR TITLE
Move value to be a property of payload, not variant

### PR DESCRIPTION
**Pull request by melindafekete**
## About the changes
Fix a minor misalignment in the AB testing doc => Value is a property of the payload, not of the variant.

### CODE REVIEW

File: website/docs/feature-flag-tutorials/use-cases/a-b-testing.md [hunk 1]
```diff
@@ -196,7 +197,7 @@ Here is an example of a payload that is returned from Google Analytics that incl
 
 By enabling impression data for your feature flag and listening to events within your application code, you can leverage this data flowing to your integrated analytics tools to make informed decisions faster and adjust your strategies based on real user behavior.
 
-### Rollout the Winning Variant to All Users
+### Roll out the Winning Variant to All Users
```
Comment (melindafekete) [inline website/docs/feature-flag-tutorials/use-cases/a-b-testing.md:200]:
```suggestion
### Roll Out the Winning Variant to All Users
```

**Review by nnennandukwe — APPROVED** [review]
Thanks ! 🙌🏽

File: website/docs/feature-flag-tutorials/use-cases/a-b-testing.md [hunk 2]
```diff
@@ -222,13 +223,10 @@ The variants you created with Unleash would be the "arms" in the multi-bandit co
...
-2. Capture impression events in your application code
-3. Funnel the impression events captured from your application code to an external analytics tool
-4. Create [signal endpoints](/reference/signals) in Unleash and point them to your external analytics tools
-5. Create [actions](/reference/actions) in Unleash that can react to your signals
-
-Learn how to configure [actions](/reference/actions) and [signals](/reference/signals) from our documentation to get started.
+1. Collect the necessary data from each variant’s performance by enabling impression data for your feature flag.
+2. Capture impression events in your application code.
+3. Funnel the impression events captured from your application code to an external analytics tool.
+4. Use the Unleash API to dynamically to adjust the traffic for each variant based on performance.
```
Comment (melindafekete) [inline website/docs/feature-flag-tutorials/use-cases/a-b-testing.md:229]:
```suggestion
4. Use the Unleash API to dynamically adjust the traffic for each variant based on performance.
```

File: website/docs/feature-flag-tutorials/use-cases/a-b-testing.md [hunk 3]
```diff
@@ -222,13 +223,10 @@ The variants you created with Unleash would be the "arms" in the multi-bandit co
...
-2. Capture impression events in your application code
-3. Funnel the impression events captured from your application code to an external analytics tool
-4. Create [signal endpoints](/reference/signals) in Unleash and point them to your external analytics tools
-5. Create [actions](/reference/actions) in Unleash that can react to your signals
-
-Learn how to configure [actions](/reference/actions) and [signals](/reference/signals) from our documentation to get started.
+1. Collect the necessary data from each variant’s performance by enabling impression data for your feature flag.
+2. Capture impression events in your application code.
+3. Funnel the impression events captured from your application code to an external analytics tool.
+4. Use the Unleash API to dynamically adjust the traffic for each variant based on performance.
```
Comment (melindafekete) [inline website/docs/feature-flag-tutorials/use-cases/a-b-testing.md:229]: @nnennandukwe @sebastian-bury I kept it high level, do you think it works?

Comment (nnennandukwe) [reply website/docs/feature-flag-tutorials/use-cases/a-b-testing.md:229]: Yeah I like this. Keeping it general gives them room to decide how they want to go out about the details specific to their needs, while pointing them in the right direction to at least get set up / started with the features Unleash has for it.

**Review by nnennandukwe — APPROVED** [review]

**Pull request merged**